packages/tools/update-readme-contributors.ts

```
@@ -12,22 +12,27 @@ const readmePath = `${rootDir}/README.md`;
 const { insertContentIntoFile } = require('./tool-utils.js');
 
 async function gitHubContributors(page: number): Promise<Contributor[]> {
-	return new Promise((resolve, reject) => {
-		request.get({
-			url: `https://api.github.com/repos/laurent22/joplin/contributors${page ? `?page=${page}` : ''}`,
-			json: true,
+async function gitHubContributors(page: number): Promise<Contributor[]> {
```

ESLint

[error] Expected indentation of 1 tab but found 0.

(`@typescript-eslint/indent`)

---

[error] Expected indentation of 1 tab but found 0.

(indent)